pid:
  kp: 0.5
  ki: 0.05
  kd: 0.001
pure_pursuit:
  mode: fixed  # 1) [fixed] lookahead distance 2) [variable] k_pre * v
  lookahead_distance: 12.0 # [m]
  k_pre: 1.5
control_switch:
  longitudinal: true # Start: true; Shutdown: false
  lateral: true
longitudinal_mode: 3 # 1: constant speed 2: planned speed 3: desired distance
desired_speed: 5 # km/h TODO: to check
desired_distance: 5 # m
lon_controller_id: 1 # 1: pid
lat_controller_id: 2 # 1: pure pursuit, 2: lqr 